src/ui/pages/IdentifierDetails/components/IdetifierDetailModal/IdentifierDetailModal.scss

```diff
@@ -0,0 +1,175 @@
+.identifier-detail-modal {
+    .page-header {
+        ion-toolbar.ios {
+            padding-left: var(--ion-safe-area-left);
+            padding-right: var(--ion-safe-area-right);
+        }
+    }
+
+    .prop-explain {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        margin: 0.75rem 0;
+
+        h2 {
+            font-weight: 700;
+        }
+
+        p {
+            color: var(--ion-color-action-blue);
+            font-size: 1rem;
+            line-height: 1.375rem;
+        }
+    }
+
+    .list-item {
+        ion-item {
+            display: flex;
+            align-items: center;
+
+            &::part(native) {
+                margin: 0.75rem 0;
+            }
+
+            &:first-child {
+                margin-top: 0;
+            }
+        }
+
+        .user-label {
+            padding: 0.25rem 0.5rem;
+            background: var(--ion-color-primary-gradient);
+            border-radius: 0.25rem;
+            display: flex;
+            align-items: center;
+            margin-left: 0.5rem;
+
+            ion-icon {
+                margin-right: 0.25rem;
+            }
+
+            span {
+                font-weight: 400;
+                font-size: 0.75rem;
+                line-height: 1rem;
+            }
+        }
+
+        .bottom-text {
+            text-align: center;
+            color: rgba(54, 60, 74, 0.5);
```

Comment:
Please replace hardcoded values for colors. 

Line 61 should be `--ion-color-secondary-rgb`.

Check also line 91 and 116.